src/main/features/ccj/views/paid-amount.njk

```diff
@@ -22,21 +22,21 @@
       {% set selection = form.valueFor('option').value %}
       <div class="form-group {% if error %} form-group-error {% endif %}">
         <fieldset id="option[label]">
-          <legend class="visually-hidden">{{ t("Has the defendant paid some of the amount owed?") }}</legend>
+          <legend class="visually-hidden">{{ t('Has the defendant paid some of the amount owed?') }}</legend>
 
           {{ radioOption(label = 'Yes', name = 'option', form = form, value = 'yes', checked = (selection == 'yes')) }}
 
           <div class="panel panel-border-narrow js-hidden" id="option-yes">
 
             <p>
               {{
-                t("Total amount payable by defendant is %s", {
+                t('Total amount payable by defendant is %s', {
                     postProcess: 'sprintf', sprintf: [ totalAmount | numeral ]
                   }
                 )
               }}
             </p>
-
+            <input type="hidden" name="claimedAmount" value="{{totalAmount}}" />
```

Comment:
why is there a hidden amount on the page?

Reply:
@robertparkinson It's needed for validation -- we need to check that the use hasn't entered a value greater that the claim total amount.

Reply:
I'm not familiar with the business logic of this feature so I hope this information is ok for the user to see/know incase someone starts looking at the source of the page and sees this value and if it is ok for the user to see it maybe it should be displayed to the user so they know not to go above that value?

Reply:
It is displayed to the user as a static text in label :)